Misc/MachineSetup/Azure_VM_Setup_Windows.ps1

```diff
@@ -248,7 +248,7 @@ function InstallAzureUtilities(){
     # Install Azure Storage Explorer
     Write-Output "Install Azure Utilities: Azure Storage Explorer"
     $LocalPath = [IO.Path]::GetTempFileName() + ".zip"
-    $web_client.DownloadFile("http://download-codeplex.sec.s-msft.com/Download/Release?ProjectName=azurestorageexplorer&DownloadId=891668&FileTime=130530255103730000&Build=20959", $LocalPath)
+    $web_client.DownloadFile("http://download-codeplex.sec.s-msft.com/Download/Release?ProjectName=azurestorageexplorer&DownloadId=891668&FileTime=130530255103730000&Build=21018", $LocalPath)
```

Comment:
This is the bit that fixes the downloading.  Other changes are some fwlinks I made and never bothered to pull from my branch.  now is a better time than any to do those as well.